covid/transform.py

```diff
@@ -521,6 +521,33 @@ def transform_covidtracking_data(df):
     return df
 
 
+def transform_cdc_ili_data(df):
+    # Use the first row of the index as the index names.
+    df.index.names = df.index[0]
+
+    # Drop the row containing the labels, and set a new multi-index using the region, year, and week.
+    df = df.iloc[1:].reset_index()
+
+    # Validate that the only region type is states to sanity check data.
+    assert set(df["REGION TYPE"].unique()) == {"States"}
+
+    # Create a new column that combines the `YEAR` and `WEEK` column.
+    # Note: The `-6` sets the date field to the start of the weekend (Saturday) for each week. We also subtract 1 week
+    #   to start the weeks at zero and ensure that they align with `https://www.epochconverter.com/weeks/2020`.
+    df[DATE_SOURCE_FIELD] = pd.to_datetime(
```

Comment:
Is this converting to the last day (Saturday?) of the current week, as previously discussed? Or is it converting to the Saturday of the previous week?

Reply:
I had to subtract 1 for some reason to get it to align with the linked week number tracker and Ryan's current sheet.

Reply:
I think because the CDC's week numbers are 1-indexed instead of 0-indexed

Reply:
Makes sense!